1차/k번째수_류동준.py

```diff
@@ -0,0 +1,6 @@
+def solution(array, commands):
+    answer = []
+    for i,j,k in commands:
+        tmp = sorted(array[i-1:j])
+        answer.append(tmp[k-1])
+    return answer
```

Comment:
```suggestion
    return [sorted(array[i-1:j])[k-1] for i,j,k in commands]
```
- 형태로 코드를 줄일 수 있을 것 같아요
- 그리고 i, j, k 변수 보다는 의미 있는 이름으로 변경하는게 좋을 것 같아요.